### ISSUE TITLE
Translate and localize the guide?

### ISSUE THREAD

**Issue by zenmonkeykstop** [OpenNewsLabs/field-guide-security-training-newsroom#85]
The initial version of the guide was written with North American English-speaking newsrooms in mind, but digital security is important for journalists working in all communities and languages. If you can provide translations of the existing guide resources, or if you'd like to add lesson plans covering topics more relevant to your community, please grab this issue or create a new one!

**Comment by deimidis** [comment]
Which will be the best way to do this job? Should I create a fork and work in it?

**Comment by zenmonkeykstop** [comment]
Please do!

**Comment by deimidis** [comment]
We were discussing with Ryan by email (i sent an email before finding this issue). Some clarifications:

It will be clear that translation will need to have a localization, and by this we mean that should be clear that each country/region need to verify locally if the tools and reccomendations in the guide is enough and good for the region.

Adding a clarification paragraph or an appendix with local news and recommendations can be a solution.

**Comment by ryanpitts** [comment]
Sounds awesome @deimidis!

One thing we've been trying to think through is how to account for the legal and cultural contexts around security in different countries. That seems really important here, so wondering if you have any thoughts. Is there a particular audience you have in mind? Maybe you have local security experience already!

_Super_ excited to see translation work on these lesson plans. This feels like something worth at least raising early on, to see if there's any kind of support we might be able to provide.

**Comment by ryanpitts** [comment]
Haha, that's what I get for having a tab open but not refreshing before leaving a comment. Awesome that you're already thinking about this, so please let us know if there's anything we can do to help figure out the right approach, or connect with folks who can help do that kind of localized verification.

**Comment by zenmonkeykstop** [comment]
Thinking out loud coz I'm not in a writing place)
There isn't a 1-1 or onto mapping between countries and languages - for example, a guide for Mexico will need to account for different threat models than a guide for Spain, but a Spanish-language guide directly translated from the existing one would be useful for Latinx communities in the US, as the threat model would be largely the same. (And even within countries different communities may be subject to different levels of threat.)

Individual tools are pretty much designed around a single threat model, so a lot of lesson plans covering tools and processes are directly translatable if that threat model holds in the region in  question. It might be worthwhile adding something to the lesson plans about assumptions that the tool or process makes. (eg. "This tool assumes that adversaries do not have access to cellular network data," or "this procedure assumes that crossing a border with a wiped phone will not raise suspicion").
 
IMO the most important stuff to localize will probably be discussions of pathways and organizational preparedness.

**Comment by deimidis** [comment]
Hello, sorry for the silence.

I'm thinking to propose a session at [Mozilla Festival](https://mozillafestival.org/proposals) to work on  this "translation". It will be great if some of you can join to think and make it there.

My current idea:

* Think about problems in "translate" this work (different threat models but at the same time some tools will be there in any threat model)
* Maybe a tutorial/manual on how to create a threat model for each country/region (region inside a country)
* A way to go from that threat model to create you own training material for Newsrooms

I will begin to work in the proposal and share it here (or if here it's not the place, and you are interested, let me know where)

**Comment by amandabee** [comment]
@deimidis I just emailed you -- let's talk about MozFest and Media Party?

**Comment by cecileriallot** [comment]
Hi everyone,

I got in touch with Ryan by email to let him know about my interest to translate the guidebook into French. As already mentioned, I understand that the guide is currently only designed with North American newsrooms in mind, but I think that a French version might be useful for some French-speaking communities in Quebec, both local and immigrant. 

Hopefully, some other French-speaking folks from other world's regions might also be interested in sharing their insight or some training materials that are best suited to the needs of their respective countries. 

I'm not a security expert, but I could still look for some news articles and tutorials relevant to European French-speaking newsrooms if that can help.

**Comment by amandabee** [comment]
That's great! Let us know how what you need from us!

**Comment by ryanpitts** [comment]
Yes, thank you so much @cecileriallot! I just opened [a tracking issue](https://github.com/OpenNewsLabs/field-guide-security-training-newsroom/issues/109) specifically for the French translation—I think it will simplify things to have one spot for any questions or coordination work that might help. This will be such an awesome contribution!

**Comment by cecileriallot** [comment]
Thanks Amanda! I will certainly get back to you later to seek your advice on some potential additional articles and tutorials.

**Comment by cecileriallot** [comment]
Thank you Ryan! I'll post any update in the new thread you created from now on.
